openpype/hosts/max/api/plugin.py

```diff
@@ -197,6 +274,28 @@ def create_instance_node(node):
         return node
 
 
+class MaxTyFlowDataCreatorBase(MaxCreatorBase):
+    @staticmethod
+    def create_instance_node(node):
+        """Create instance node.
+        If the supplied node is existing node, it will be used to hold the
+        instance, otherwise new node of type Dummy will be created.
+        Args:
+            node (rt.MXSWrapperBase, str): Node or node name to use.
+        Returns:
+            instance
+        """
+        if isinstance(node, str):
```

Comment:
It is more likely the API's issue. Sometimes the node can be successfully converted as string from max object, so that's why we need to make sure it's string value. But I agree we need to raise some errors on that.